_posts/2020-03-23-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Thursday March 5, 2020
+Attended: Franklin Feingold, Melanie Ganz, Guiomar Niso, Robert Oostenveld
+
+<!--more-->
+
+### Executive Summary
+
+We reviewed the action items from the previous two meetings. We discussed the framework for the BIDS community slides and how this information can be formatted and designed for future extensions and derived versions. We will draft an approach for organizing BIDS related meetings to pilot at NRM and OHBM in June. 
+
+We also have a [list of channels](https://docs.google.com/spreadsheets/d/16SAGK3zG93WM2EWuoZDcRIC7ygPc5b7PDNGpFyC3obA/edit#gid=0) to share BIDS related information on. The have a plan for handling non-BIDS related information on our channels is to share on our [google group](https://groups.google.com/forum/#!forum/bids-discussion).
+
+The attending members of the Steering Group gave approval for moving forward with completing the [Genetic Information BEP](https://github.com/bids-standard/bids-specification/pull/395). This will solidify the process for incorporating BEPs moving forward. The process will be: 1) merging in the BEP constructed examples to [BIDS-examples](https://github.com/bids-standard/bids-examples), 2) merge the validator extension into [BIDS-validator](https://github.com/bids-standard/bids-validator), and 3) merge the BEP into [BIDS-specification](https://github.com/bids-standard/bids-specification)
+
+The Steering Group gave direction on handling author lists on Zenodo - make it a single author: “BIDS Contributors Group” as defined in the [governance document](https://docs.google.com/document/d/1R-J2lL9V_wIkYhye4zH-feyl4P4J8NyO40rIYyY141o/edit). 
+
+### Action Items
+
+| Action Item |
+| -------- |
+| GN: draft tweets for OHBMX twitter presentation    |
+| Write INCF to determine if they have a list of related standard initiatives |
+| Initialize Google Slides to start iterating on our BIDS community slides |
+| Draft specification vs ecosystem for website-about section |
+| Guidelines for organizing a BIDS information session |
+| Draft BIDS interest group information and process |
+| Codify rules for making a repository under the BIDS standard GitHub organization |
+| Share Steering Meeting: executive summary, action items, and minutes to bids-website |
```

Comment:
I think this table should rather be an unordered list. and the table header can be deleted (it's already the subsection heading on line 24)

```Markdown

- GN: draft tweets
- Write INCF
- ...

```